Modules/xSharePoint/DSCResources/MSFT_xSPCacheAccounts/MSFT_xSPCacheAccounts.psm1

```diff
@@ -4,9 +4,10 @@ function Get-TargetResource
     [OutputType([System.Collections.Hashtable])]
     param
     (
-        [parameter(Mandatory = $true)] [System.String] $WebAppUrl,
-        [parameter(Mandatory = $true)] [System.String] $SuperUserAlias,
-        [parameter(Mandatory = $true)] [System.String] $SuperReaderAlias,
+        [parameter(Mandatory = $true)]  [System.String]  $WebAppUrl,
+        [parameter(Mandatory = $true)]  [System.String]  $SuperUserAlias,
+        [parameter(Mandatory = $true)]  [System.String]  $SuperReaderAlias,
+        [parameter(Mandatory = $false)] [System.Boolean] $SetWebAppPolicy = $true,
```

Comment:
I do because that's the behavior we have currently and I don't want to change the behavior for people who are already using xSharePoint. The ones who want to manage the web app policy through the new resource can implement this property to set it to false and get what they want as a behavior moving forward.